src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordWithOcrTest.java

```diff
@@ -72,9 +111,25 @@ void should_update_case_with_ocr_data() {
         verifySuccessResponse(response, caseData);
     }
 
+    @DisplayName("Should successfully pass if exception record already attached to another case")
+    @Test
+    void should_fail_if_record_already_attached_to_another_case() throws JsonProcessingException {
```

Comment:
Display name not matching test name.

Perhaps remove the annotation completely, it adds no value imho.